src/screencastPanel.ts

```diff
@@ -137,6 +139,19 @@ export class ScreencastPanel {
         return view.render();
     }
 
+    private onSocketTelemetry(message: string) {
+        const telemetry: TelemetryData = JSON.parse(message) as TelemetryData;
+        if (telemetry.event !== 'screencast') {
+            return;
+        }
+
+        const properties: ITelemetryProps = {};
+        properties[telemetry.name] = JSON.stringify(telemetry.data);
```

Comment:
@hkal I see this is based on how we submit errors, but will we be able to query / create summaries based on these stringified pieces of data?

```
devtools/Screencast.Emulation: {"Screencast.Emulation":"{\"event\":\"forcedColors\",\"value\":\"active\"}"}, undefined
```

Reply:
Similar comment.  Would prefer to have the data formatted like this:
```
devtools/Screencast.Emulation: {"event":"forcedColors","value":"active"}, undefined
```